src/libopensc/pkcs15-piv.c

```diff
@@ -673,23 +702,25 @@ static int sc_pkcs15emu_piv_init(sc_pkcs15_card_t *p15card)
 		
 		r = sc_pkcs15emu_object_add(p15card, SC_PKCS15_TYPE_DATA_OBJECT, 
 			&obj_obj, &obj_info); 
+//debug fprintf(stderr,"%s:%d r=%d i=%d label=%s\n", __FILE__, __LINE__, r, i, objects[i].label);
+
```

Comment:
Can we remove this debugging comment, or is it still useful to convert it to normal debug log?